cyhy/mailer/cli.py

```diff
@@ -1060,6 +1145,32 @@ def main():
         )
         return 1
 
+    # Parse out the CSA email mapping, if present.
+    csa_email_file = args["--csa-email-file"]
+    if csa_email_file is not None:
```

Comment:
Can `csa_email_file` ever be `None` since we are setting a default value for it in the docopt options above(`/run/secrets/csa_emails.yml`)?
https://github.com/cisagov/cyhy-mailer/blob/8aecdbfef831a091aab86a5c189d7e5a8aef9a5d/cyhy/mailer/cli.py#L34-L50

Reply:
Please see commit 56b729e4f7a7432e2a0faae37bad1d0ea6a3aaca.

Reply:
Please see commit [ffccc70](https://github.com/cisagov/cyhy-mailer/pull/101/commits/ffccc70d93e614d8e4dbe5eb7ff8ae77fce8acfe).  I tested by doing:
- A full CyHy reports dry run where the CSA email YAML was specified
- A full CyHy reports dry run without specifying the CSA email YAML

Reply:
🌶️ 💩 🔥